ReactiveXComponentTest/XComponentTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.IO;
+using System.Net.Http;
+using NFluent;
+using NSubstitute;
+using NUnit.Framework;
+using ReactiveXComponent;
+
+namespace ReactiveXComponentTest
+{
+    [TestFixture]
+    public class XComponentTest
+    {
+        private IXCSessionFactory _sessionFactory;
+        private XComponentApi _xcApi;
```

Comment:
isolate your test. As you're testing XComponentApi, it should not have been defines as instance variable